Cargo.toml

```diff
@@ -39,6 +39,7 @@ default-members = [
     "libparsec/crates/tests_macros",
     "libparsec/crates/types",
 ]
+resolver = "2"
```

Comment:
According to the [docs](https://doc.rust-lang.org/edition-guide/rust-2021/default-cargo-resolver.html), using `edition = 2021` implies `resolver = 2`, so this line is redundant

Reply:
Still in the doc:

> If you are using a [virtual workspace](https://doc.rust-lang.org/cargo/reference/workspaces.html#virtual-workspace), 
> you will still need to explicitly set the [resolver field](https://doc.rust-lang.org/cargo/reference/resolver.html#resolver-versions) in the `[workspace]` definition if you want to opt-in to the new resolver.

Reply:
So maybe we can add a comment in case someone has the same doubt? 
```toml
# Set the resolver explicitly becase we are in a [workspace] definition
# See: https://doc.rust-lang.org/cargo/reference/workspaces.html#virtual-workspace
resolver = "2"
```

Reply:
It will warns you anyway if you forget it

Reply:
Really? well in that case, no need for the comment I guess